pkg/ulimit/ulimit_test.go

```diff
@@ -2,6 +2,13 @@ package ulimit
 
 import "testing"
 
+func TestParseValid(t *testing.T) {
+	u1 := &Ulimit{"nofile", 1024, 512}
+	if u2, _ := Parse("nofile=512:1024"); u1 == u2 {
```

Comment:
I'm not sure if it is right. You comparing pointers to `Ulimit` which obviously not equal and that's why your condition `u1 == u2` is true. You should use `*u1 != *u2` I think.